css/components/Toast/toast.css

```diff
@@ -0,0 +1,52 @@
+.toast {
+  background: white;
+  border-radius: 0.5rem;
+  box-shadow: 0.1rem 0.1rem 0.5rem var(--COLOR-SLATE-300);
+  max-width: 30rem;
+  min-width: 30rem;
+  padding: 1.5rem;
+  transition: opacity 0.4s;
+}
+
+.toast-cross {
+  border: none;
+  border-radius: 50%;
+  color: var(--COLOR-SLATE-800);
+  height: 2rem;
+  width: 2rem;
+}
+
+.toast-cross:hover {
+  background: var(--COLOR-SLATE-300);
+  color: var(--COLOR-SLATE-600);
+}
+
+.toast-head {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: var(--COLOR-SLATE-800);
+  margin-bottom: 1rem;
+}
+
+.toast-link {
+  color: var(--COLOR-BRAND-DARK);
+  font-weight: bold;
+}
+
+.toast-link:hover {
+  color: var(--COLOR-BRAND);
+}
+
+.toast-msg {
+  font-size: 1.4rem;
+  color: var(--COLOR-SLATE-700);
+  margin-top: 1rem;
+}
+
+.hide-toast {
+  opacity: 0;
```

Comment:
You are using opacity:0 to hide the toast. I think display none would be a better option.

Reply:
On `display: none`, `transition` property does not work. That's why to add some transition, I am using opacity.

Reply:
ok fine then no issues

Reply:
Hey on the same issue , 
if you use opacity "0" I think it will still be clickable, either try pointer events none, or scale(0) , or remove element after transition complete through java script.

![image](https://user-images.githubusercontent.com/90306959/153236256-bf103d88-ad31-439e-961f-77fa1aee996c.png)

Reply:
@Vishnu-Aithal Thanks for letting me know. 

For now, I am using `display: none`, for now, I am not using any animation. I have decided to add when Vikram will teach us that.